apps/rush-lib/src/data/RushConfiguration.ts

```diff
@@ -154,10 +156,13 @@ export default class RushConfiguration {
     const expectedRushVersion: string = rushConfigurationJson.rushVersion;
     // If the version is missing or malformed, fall through and let the schema handle it.
     if (expectedRushVersion && semver.valid(expectedRushVersion)) {
-      if (semver.eq(rushVersion, expectedRushVersion)) {
-        throw new Error(`Your rush tool is version ${rushVersion}, but rush.json`
-          + ` requires version ${rushConfigurationJson.rushVersion}. To upgrade,`
-          + ` run "npm install @microsoft/rush -g".`);
+      if (semver.lt(rushVersion, expectedRushVersion)) {
+        throw new Error(`Your rush tool is version ${rushVersion}, but rush.json ` +
+          `requires version ${rushConfigurationJson.rushVersion}. To upgrade, ` +
+          `run "npm install @microsoft/rush -g".`);
+      } else if (semver.lt(expectedRushVersion, MINIMUM_SUPPORTED_RUSH_JSON_VERSION)) {
```

Comment:
This is not really needed yet.

Reply:
It doesn't hurt anything to include it, though.

Reply:
Doesn't hurt anything to include it.

---
In reply to: [147537083](https://github.com/Microsoft/web-build-tools/pull/417#discussion_r147537083) [](ancestors = 147537083)